test/local/metrics/events.js

```diff
@@ -8,13 +8,21 @@ const assert = require('insist')
 const sinon = require('sinon')
 const log = {
   activityEvent: sinon.spy(),
+  amplitudeEvent: sinon.spy(),
   error: sinon.spy(),
   flowEvent: sinon.spy()
 }
 const events = require('../../../lib/metrics/events')(log)
 const mocks = require('../../mocks')
 
 describe('metrics/events', () => {
+  afterEach(() => {
+    log.activityEvent.reset()
+    log.amplitudeEvent.reset()
+    log.error.reset()
+    log.flowEvent.reset()
```

Comment:
Saw that `Date.now.restore()` was used in some finally blocks, could they be moved here?

Reply:
`Date.now.restore()` would throw for the tests that don't stub it (which is most of them). I could put it in a try/catch I guess, or check for the existence of `restore` before calling it but tbh I don't think it belongs in `afterEach` if it's specific to certain tests.